wiki/Tournaments/KEL/4/en.md

```diff
@@ -0,0 +1,222 @@
+  2. [cubesato - Shigatsu no Ame (Sinnoh) [Expert]](https://osu.ppy.sh/beatmapsets/1516919#osu/3105402)
+  3. [cubesato - Ssdfsfsdhigatsu no Ame (Sinnoh) [Expert]](https://osu.ppy.sh/beatmapsets/1516919#osu/3105402)
+- Hard Rock
+  1. [Midnight Grand Orchestra - Igniter (RafaelXDP) [Light Speed]](https://osu.ppy.sh/beatmapsets/2276636#osu/4851845)
+  2. [boa - Duvet (a night parrot) [Confused Reality]](https://osu.ppy.sh/beatmapsets/2285299#osu/4874635)
+  3. [boa - Duvesdfsfsfdt (a night parrot) [Confused Reality]](https://osu.ppy.sh/beatmapsets/2285299#osu/4874635)
+- Double Time
+  1. [Official HIGE DANdism - Same Blue (Kyouren) [f2alon // Kyouren // Kazuha's Insane]](https://osu.ppy.sh/beatmapsets/2258440#osu/4807674)
+  2. [A-One feat. ayaponzu* - Justice Monster (Chizu-Kun) [Insane]](https://osu.ppy.sh/beatmapsets/1513750#osu/3105221)
+  3. [kors k - SigSig (AJT) [Nisan's Insane]](https://osu.ppy.sh/beatmapsets/1615650#osu/3298595)
+- Tiebreaker
+  1. **[Down - Kizan (Down) [Slayer]](https://osu.ppy.sh/beatmapsets/2085347#osu/4368642)**
+
+### Qualifiers
+
+- No Mod
+  1. [The Offspring - The Future Is Now (piroshki) [Imminence]](https://osu.ppy.sh/beatmapsets/2119418#osu/4452129)
+  2. [technoplanet - Intuition (wakaba) [Sententialism]](https://osu.ppy.sh/beatmapsets/1824775#osu/3744646)
+  3. [ZUTOMAYO - TAIDADA (Petal) [RESONATION]](https://osu.ppy.sh/beatmapsets/2270896#osu/4836943)
+  4. [Cansol - 1,2,3,4! (Acylica) [geunssaem]](https://osu.ppy.sh/beatmapsets/2021545#osu/4210079)
+- Hidden
+  1. [UKRampage - Yumenoshima feat. mami (Gillstar) [Gill & Sharu's Dreamscape]](https://osu.ppy.sh/beatmapsets/1644513#osu/3356838)
+  2. [Yasui Yousuke - YO-KAI Disco [MEIKAI ARRANGE VERSION] ([Keqing]) [aww shidd]](https://osu.ppy.sh/beatmapsets/1485831#osu/3046747)
+- Hard Rock
+  1. [Akatsuki Records - Treasure Cirno (Villdjack) [Stacked]](https://osu.ppy.sh/beatmapsets/1498792#osu/3071745)
+  2. [Hagall x 152Hz - In midst of time feat. towatari (Cut Ver.) (sorciere) [Exploring the Unknown]](https://osu.ppy.sh/beatmapsets/1893886#osu/3902759)
+- Double Time
+  1. [Asakura Hinata - Yukitoki (Amateurre) [Gelidium]](https://osu.ppy.sh/beatmapsets/1569543#osu/3205099)
+  2. [Pendulum - Hold Your Colour (GRR SNARL GROWL) [Hold Your Growlr]](https://osu.ppy.sh/beatmapsets/1834500#osu/3766173) -->
+
+## Ruleset
+
+### General information
+
+1. KEL4 is a 1v1 multi-bracket LAN tournament (single, later double elimination) played within the osu! stable client.
+2. All matches will be played with ScoreV2 in Head-to-Head mode.
+3. All maps will be played with No Fail enabled, along with the appropriate mods for that map.
+4. Each stage's mappool will be revealed during the official tournament stream.
+5. Special guests will be undergoing a special qualifying process separate from the normal players. Their results will be revealed once all qualifying lobbies have concluded.
```

Comment:
idk if it affects badging but is there really no more information to include? if it's just that you're sending out invites to some people at your discretion, that would be more clear than this